.pylintrc

```diff
@@ -193,7 +193,7 @@ max-nested-blocks=5
 [FORMAT]
 
 # Maximum number of characters on a single line.
-max-line-length=150
+max-line-length=160
```

Comment:
Should we be doing 120, 150, 160? Do we know what we want as a team?

Reply:
160 is what the travis currently fails at.  Let's start by making everything else consistent with that, we can discuss if we want to change separately.